ChangeLog

```diff
@@ -9,6 +9,9 @@ Release date: Undefined
 ..
   Put new features here and also in 'doc/whatsnew/2.8.rst'
 
+* Add new ``ConfusingConsecutiveElifChecker``. This optional checker emits a refactoring message (R5601 ``confusing-consecutive-elif``)
```

Comment:
```suggestion
* Add new extension ``ConfusingConsecutiveElifChecker``. This optional checker emits a refactoring message (R5601 ``confusing-consecutive-elif``)
```
Maybe add the word `extension` to be extra clear? (For the whatsnew section as well)